### PR TITLE
Support "urgent tickets" queries

### LINKED ISSUE #48: Create two new queries to include critical, high, and KEVs for urgent tickets
## Task ##

In support of BOD 22-01 requirements 10 and 11: Add two new queries to include criticals, highs, and KEVs:

- csv_get_open_urgent_tickets
- csv_get_closed_urgent_tickets

## Comments ##

The associated CSVs are generated here: https://github.com/cisagov/cyhy-reports/blob/develop/extras/create_snapshots_reports_scorecard.py#L747-L777 

The two new queries should be added here: https://github.com/cisagov/ncats-webd/blob/develop/ncats_webd/cybex_queries.py

### References ###

- Epic https://github.com/cisagov/cool-system-internal/issues/107
- Story #34 
- Current Estimated LOE: 7 hours for these tasks.

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR updates some queries that are used to create CSVs containing open or recently-closed tickets.  It allows for a new faux severity called "urgent", which includes tickets that meet at least one of the following criteria:
* KEV (Known Exploited Vulnerability) = true
* Critical (severity = 4)
* High (severity = 3)

In addition, I also added the KEV flag and ticket severity to the CSV output.

During my testing, I encountered a pre-existing bug, so I fixed that in https://github.com/cisagov/ncats-webd/commit/0b0336e2da4eb762542d185cdc51688142361d57.

## 💭 Motivation and context ##

This PR resolves https://github.com/cisagov/cyhy-system/issues/48.
This PR covers part of the work requested in https://github.com/cisagov/cyhy-system/issues/34.

## 🧪 Testing ##

I used this updated code to generate new "urgent tickets" CSVs (for both open and closed tickets) using my test data and verified that the output looked as expected.

I also tested my code by generating the existing CSVs (for critical-severity tickets and high-severity tickets) and the urgent tickets CSVs, using Production data and verified that the output looked as expected.  There were no unexpected changes to the existing CSVs (which will continue to be used by the CyHy Dashboard).

To generate my testing CSVs, I wrote some test code similar to [this](https://github.com/cisagov/cyhy-reports/blob/develop/extras/create_snapshots_reports_scorecard.py#L747-L777).

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Comment by mcdonnnj** [comment]
@dav3r Would you manually run `black` against this source file and push that update? The formatting is inconsistent (and in one case offensive to my eyes).

**Comment by dav3r** [comment]
> @dav3r Would you manually run `black` against this source file and push that update? The formatting is inconsistent (and in one case offensive to my eyes).

@mcdonnnj No problem- done in https://github.com/cisagov/ncats-webd/pull/27/commits/ecbabc25eceb0ee578a87b98cf7810db274fce7e.  Command used was:
```
black -t py27 ./cybex_queries.py
```

File: ncats_webd/cybex_queries.py [hunk 1]
```diff
@@ -18,25 +18,47 @@ def get_open_tickets_dataframe(db, ticket_severity):
...
+        #   KEV (Known Exploited Vulnerability) = true
+        #   Critical (severity = 4)
+        #   High (severity = 3)
+        tix = db.TicketDoc.find(
+            {
+                "source": "nessus",
+                "$or": [{"details.kev": True}, {"details.severity": {"$gte": 3}}],
+                "false_positive": False,
+                "owner": {"$in": fed_executive_owners},
+                "open": True,
```
Comment (mcdonnnj) [inline ncats_webd/cybex_queries.py]: Just a thought.

```suggestion
                "$or": [{"details.kev": True}, {"details.severity": {"$gte": 3}}],
                "false_positive": False,
                "open": True,
                "owner": {"$in": fed_executive_owners},
                "source": "nessus",
```

File: ncats_webd/cybex_queries.py [hunk 2]
```diff
@@ -18,25 +18,47 @@ def get_open_tickets_dataframe(db, ticket_severity):
...
+        )
+    else:
+        # Treat ticket_severity normally
+        tix = db.TicketDoc.find(
+            {
+                "source": "nessus",
+                "details.severity": ticket_severity,
+                "false_positive": False,
+                "owner": {"$in": fed_executive_owners},
+                "open": True,
```
Comment (mcdonnnj) [inline ncats_webd/cybex_queries.py]: Same thought.

```suggestion
                "details.severity": ticket_severity,
                "false_positive": False,
                "open": True,
                "owner": {"$in": fed_executive_owners},
                "source": "nessus",
```

File: ncats_webd/cybex_queries.py [hunk 3]
```diff
@@ -85,25 +107,47 @@ def get_closed_tickets_dataframe(db, ticket_severity):
...
+        #   KEV (Known Exploited Vulnerability) = true
+        #   Critical (severity = 4)
+        #   High (severity = 3)
+        tix = db.TicketDoc.find(
+            {
+                "source": "nessus",
+                "time_closed": {"$gte": closed_since_date},
+                "$or": [{"details.kev": True}, {"details.severity": {"$gte": 3}}],
+                "owner": {"$in": fed_executive_owners},
+                "open": False,
```
Comment (mcdonnnj) [inline ncats_webd/cybex_queries.py]:
```suggestion
                "$or": [{"details.kev": True}, {"details.severity": {"$gte": 3}}],
                "open": False,
                "owner": {"$in": fed_executive_owners},
                "source": "nessus",
                "time_closed": {"$gte": closed_since_date},
```

File: ncats_webd/cybex_queries.py [hunk 4]
```diff
@@ -85,25 +107,47 @@ def get_closed_tickets_dataframe(db, ticket_severity):
...
+        )
+    else:
+        # Treat ticket_severity normally
+        tix = db.TicketDoc.find(
+            {
+                "source": "nessus",
+                "time_closed": {"$gte": closed_since_date},
+                "details.severity": ticket_severity,
+                "owner": {"$in": fed_executive_owners},
+                "open": False,
```
Comment (mcdonnnj) [inline ncats_webd/cybex_queries.py]:
```suggestion
                "details.severity": ticket_severity,
                "open": False,
                "owner": {"$in": fed_executive_owners},
                "source": "nessus",
                "time_closed": {"$gte": closed_since_date},
```

**Review by mcdonnnj — APPROVED** [review]
I had suggestions for sorting the keys in the dictionaries used in the `find()` calls but otherwise LGTM.

**Pull request merged**